extensions/common/context.cc

```diff
@@ -338,6 +339,19 @@ void populateExtendedRequestInfo(RequestInfo* request_info) {
   getValue({"destination", "address"}, &request_info->destination_address);
   getValue({"source", "port"}, &request_info->source_port);
   getValue({"connection_id"}, &request_info->connection_id);
+  getValue({"route_name"}, &request_info->route_name);
+  getValue({"upstream", "address"}, &request_info->upstream_host);
+  getValue({"upstream", "cluster"}, &request_info->upstream_cluster);
```

Comment:
When we have a 404 NR or in the other case 503 UC or UH it will not leave garbage value, right?

Reply:
it should be empty in that case